views/header.hjs

```diff
@@ -16,6 +16,14 @@
 <meta charset="utf-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
 
+  <title>R-hub package builder</title>
+<meta property="og:title" content="R-hub package builder">
+<meta property="og:type" content="website">
+<meta property="description" content="R package builder, by the R-hub project of the R Consortium, easing the R package development process.">
+<meta property="og:description" content="R package builder, by the R-hub project of the R Consortium, easing the R package development process.">
+<meta name="twitter:card" content="summary">
+<meta name="twitter:image" content="https://builder.r-hub.io/img/rhub-square-white.png" >
```

Comment:
Should this be the canonical place for the logo(s)? Because then we might as well put all of them here....

Reply:
Good thinking! But on other subdomains it can be good to have the logo in the site too for other styling 🤔